mongodb/driver.go

```diff
@@ -94,16 +97,24 @@ type MongoConnectionConfig struct {
 	TLSConnectionConfig
 }
 
-func (m *MongoConnectionConfig) GetConnectionURI() string {
+func (m *MongoConnectionConfig) GetConnectionURI() (string, error) {
 	var connectionString string
+	endpoint := m.ClusterEndpoint
+
+	matches, _ := regexp.MatchString(endpointRegex, endpoint)
```

Comment:
The error that MatchString might return ought to be handled.

Reply:
Ok, I thought I would ignore it because the regex is a constant and known value that should not cause an error but will handle it 👍